app/assets/stylesheets/provider/admin/cms/_cms_intro.scss

```diff
@@ -80,22 +80,27 @@ $glow-color: $highlight-color;
 
 #cms-sidebar {
   li.glowing.active, li.glowing {
-    color: $glow-color !important;
+    background-color: $wildSandGray;
   }
 
   input.glowing {
+    background-color: $wildSandGray;
     color: $glow-color !important;
```

Comment:
Is the glow-colour doing anything?

Reply:
while you're at it: `$glow-color: $highlight-color;` should also be defined in colors.

Reply:
done

Reply:
yes, squash to 1 commit and good to merge